Automattic/wp-calypso
# fix(hosting-overview-refinements): anchors for WP and PHP versions

Related to https://github.com/Automattic/dotcom-forge/issues/8653

## Proposed Changes
When the “Server Settings” tab is active, clicking on the WP / PHP versions doesn’t do anything. The expected behaviour is that we’d auto-scroll users to the Web server settings box.

## Testing Instructions
1. Open `/sites`
2. Select Atomic site
3. Click on WP version
4. Assert that you are landed to "Server Settings" with autoscroll to WP version settings
5. Click on PHP
6. Assert that you auto scrolled to the PHP version (on big screen it scrolles literally to the same place as click on WP, but anyway we see the scroll, so we have a feedback from UI, and it's the main point, to help a user understand that PHP is located in front of them).
